samples/apps/http-apps/README.md

```diff
@@ -0,0 +1,231 @@
+# HTTP Protocol Sample Device|Discovery apps
+
+This directory provides implementations of IoT devices and a discovery service that can be used to test the Akri HTTP Protocol Broker.
+
+This directory includes an alternative gRPC implementation of the Akri HTTP Protocol gRPC Broker and a Client too.
+
+## Environment
+
+```bash
+export REGISTRY="ghcr.io"
+export USER=[[GITHUB-USER]]
+export PREFIX="http-apps" # Or ...
+export TAG=$(git rev-parse HEAD) # Or ...
```

Comment:
use version.txt for TAG

Reply:
I went with `v1` for simplicity (and before seeing this suggestion). Hope Ok?

Fixed: https://github.com/deislabs/akri/pull/135/commits/60620b61185473dab04cd2fb77a3344905a2825b